src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/JitHelper.cs

```diff
@@ -198,10 +198,10 @@ public static void GetEntryPoint(TypeSystemContext context, ReadyToRunHelper id,
                     break;
 
                 case ReadyToRunHelper.DblRem:
-                    mangledName = "RhpDblRem";
+                    mangledName = "fmod";
                     break;
                 case ReadyToRunHelper.FltRem:
-                    mangledName = "RhpFltRem";
+                    mangledName = "fmodf";
                     break;
 
```

Comment:
Just noticed that `DblRound/FltRound` are unhandled here (and on main), would it make sense to readd handling for them at least until they're fully removed?

Reply:
I am not sure what you mean by unhandled. Could you please elaborate?

Reply:
> I am not sure what you mean by unhandled. Could you please elaborate?

There seems to be no switch case for them.

Reply:
```suggestion

                case ReadyToRunHelper.DblRound:
                    DefType doubleType = context.GetWellKnownType(WellKnownType.Double);
                    methodDesc = context.SystemModule.GetKnownType("System", "Math").GetKnownMethod("Round",
                        new MethodSignature(MethodSignatureFlags.Static, 0, doubleType, [doubleType]));
                    break;
                case ReadyToRunHelper.FltRound:
                    DefType floatType = context.GetWellKnownType(WellKnownType.Single);
                    methodDesc = context.SystemModule.GetKnownType("System", "MathF").GetKnownMethod("Round",
                        new MethodSignature(MethodSignatureFlags.Static, 0, floatType, [floatType]));
                    break;

```
if we want to add them here.

Reply:
This mapping is native AOT specific. It does not need to handle JIT helpers that are no longer used or that are only used outside native AOT.